CreeDictionary/utils/fst_analysis_parser.py

```diff
@@ -12,6 +12,38 @@
 )
 
 
+def split_analysis(analysis: str) -> Tuple[List[str], FSTLemma, List[str]]:
```

Comment:
Not a big deal, but this mirrors the API of `str.partition()` more than `str.split()` in the Python API.